src/yardstick/tool/syft.py

```diff
@@ -38,7 +42,7 @@ def _install_from_installer(
         version: str,
         path: Optional[str] = None,
         use_cache: Optional[bool] = True,
-        **kwargs,  # pylint: disable=unused-argument
+        **kwargs,  # noqa: ARG004
```

Comment:
Is this the right `noqa`? https://docs.astral.sh/ruff/rules/unused-static-method-argument/#unused-static-method-argument-arg004

Reply:
it is -- when I remove the exception I get:
```
$ ruff check src --fix
src/yardstick/tool/syft.py:45:11: ARG004 Unused static method argument: `kwargs`
```